### ISSUE TITLE
Auto-register clients when new account created

### ISSUE THREAD

**Issue by kenrestivo** [kenrestivo/converse-hubzilla#2]
When the Converse plugin is set up, automatically register any users on this hub, if they don't have accounts with the jabber server. 

First pulling in addresses from old jappix settings, and use those if available.

Handle registration errors and edge cases and exceptions, of which there will be multitudes. (What if they're already registered? What if the account isn't available but it's not theirs? What if the bosh or server host is incorrect and you're on someone else's server by mistake? What if there are connectivity issues-- retries, etc.)

Basically, hide all the XMPP stuff from the user. But allow them to change it and/or find it so they can use other clients if they want. This is a large project.

**Comment by kenrestivo** [comment]
This project is deprecated as hubzilla now includes an xmpp converse plugin by default.
